src/main/java/com/faforever/client/mod/ModUploadTask.java

```diff
@@ -69,10 +78,20 @@ protected Void call() throws Exception {
             .zip();
       }
 
+      // retrieve information from the mod_info.lua to send to the API
+
+      ModVersion modVersionInfo = modService.extractModInfo(modPath);
+      URL repositoryURL = modVersionInfo.mod().repositoryURL();
```

Comment:
This is shadowing the field repositoryUrl now. So the field should be removed or this should set the field if it is null.

Reply:
Processed in: https://github.com/FAForever/downlords-faf-client/pull/3192/commits/57ff0b36dea0f583c9a3bf526364eb194051d2e4